### ISSUE TITLE
Copy Evaluated Math Expression

### ISSUE THREAD

**Issue by AjayRahul1** [corphish/TextTools#6]
> First of all, thank you for this app. This is something I've really wanted to make for myself using Android Studio.

I'd like to make a simple suggestion to enhance user experience.

- **Instead of replacing the text with the evaluated math expression, copy the answer onto the Android Clipboard if possible.**

### Explanation why it enhances UX
 
I would think this enhances user experience because the user types the whole expression. If this replaces text, then if the user wanted to make any changes in the math expression, he/she needs to type it all again.

I hope this helps. Thank you :)

**Comment by corphish** [comment]
It could be given as an option along with existing method of replacing the mathematical expression with its result.

**Comment by AjayRahul1** [comment]
Right. Rename current evaluate as **Evaluate Inline**. Create a new option to evaluate and copy the result and notify the user with a toast or a snackbar. That could be better I guess. I forked this repository and added the copying part myself but it's not the total functionality. So I think it's better for you to add functionality.

P.S: I am a **basic** Android Developer. So I wouldn't be able to add the code and contribute to this repository. Else I would have done the contribution. Thank you.

**Comment by corphish** [comment]
Functionality now available in https://github.com/corphish/TextTools/releases/tag/v1.3.1.
Do try and let me know. If you have any suggestion, do mention.

**Comment by AjayRahul1** [comment]
This is the **best implementation** and I can assure you that sir.

However in a few apps, there is one issue with "Append Result to the selected text".

In _WhatsApp_, this feature worked completely fine. But in _TickTick_, this worked in the way mentioned below.

> If I select 200+100 and choose that option, Then text is becoming, **"200+100200+100 = 300"**.

Thank you for considering my suggestion and adding it sir.

**Comment by corphish** [comment]
Ok so that's a different issue. Yes it looks like some apps handle PROCESS_TEXT intent differently.

For your second suggestion, are you talking about the normal (multiple option) variant or single option variant? It is not possible for the former but possible for the later.

**Comment by AjayRahul1** [comment]
![Screenshot_20240801-112036.png](https://github.com/user-attachments/assets/e3f50ed9-0a38-4a8a-aa91-2b09a7885d89)

I am talking about this **Evaluate** menu. Once we arrive at this menu by clicking "[T²] Evaluate", both the back button and clicking on the empty area around the menu cannot take me back. We definitely need to select at least one of those 3 options.

**Comment by corphish** [comment]
Should be fixed in the [1.4.0 release](https://github.com/corphish/TextTools/releases/tag/v1.4.0).
